### PR TITLE
US26824: Python SDK - input tool sample plugin

### PR DESCRIPTION
**Pull request by bkearly**

### CODE REVIEW

File: PyInputToolExample/engine.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        self.record_info_out = None
+        self.record_creator = None
+        self.output_field = None
+
+        #
+        # create your custom members here and give them default values
+        #
+        self.file_input_name = 'C:\\Users\\username\\Desktop\\PythonInputTest.csv'
+        self.file_out = 'file object'
+        self.file_reader = 'mapped file object'
```
Comment (alteryxMChadwick) [inline PyInputToolExample/engine.py:34]: Are `self.file_out` and `self.file_reader` meaningful as strings at any point in the plugin?

File: PyInputToolExample/engine.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        self.record_creator = self.record_info_out.construct_record_creator()
+        rownum = 0
+
+        #
+        # map values in file to the tool output
+        #
+        for row in self.file_reader:
+            rownum += 1
+            for index, value in enumerate(row.items()):
+                self.record_info_out[index].set_from_string(self.record_creator, value[1])
```
Comment (alteryxMChadwick) [inline PyInputToolExample/engine.py:154]: What is `value[0]`?

File: PyInputToolExample/engine.py (hunk 1 above)
Comment (bkearly) [reply PyInputToolExample/engine.py:34]: They aren't. I was a little confused as to what should be declared in __init__ and how in the case of something that's a file object. Should they be declared? If so, as what?

File: PyInputToolExample/engine.py (hunk 2 above)
Comment (bkearly) [reply PyInputToolExample/engine.py:154]: value consists of (fieldname, value) at the specified index, so value[0] is the fieldname

File: PyInputToolExample/engine.py (hunk 1 above)
Comment (alteryxMChadwick) [reply PyInputToolExample/engine.py:34]: I would argue everything should be declared in `init`. I shared the rationale some in this other comment: https://github.com/alteryx/python-sdk-samples/commit/ef76ab73f750c703eb373e390e578017290f262c#commitcomment-23214041

As for what they should be declared as, if there's not a better default, I use `None`.

Comment (bkearly) [reply PyInputToolExample/engine.py:34]: I'll change them to None. Thanks!

**Pull request merged**